test/expectations/document_highlight/argument_forwarding.exp

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
It seems that though we had `test/fixtures/argument_forwarding.rb`, there was no corresponding `.exp` - does that mean it was having no effect? If so, might there be other 'gaps' like this, and could we catch those programatically? 🤔